setup/azurecmdfiles.wxi

```diff
@@ -885,6 +885,9 @@
                         <Component Id="cmpA7B72722B1AC91C0B6B117CEAB00E3C0" Guid="*">
                             <File Id="fil40386C22E607E953293F5632EAC6393E" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.PowerBIEmbedded\Microsoft.Azure.Commands.Management.PowerBIEmbedded.dll-Help.xml" />
                         </Component>
+                        <Component Id="cmp20C58BA5A8D7263E84BE2BCB10783E8A" Guid="*">
```

Comment:
@selasagi the `.wxi` file should be regenerated since you are adding a couple new assemblies to the `PowerBI` project